packages/@aws-cdk/assertions/README.md

```diff
@@ -103,6 +103,17 @@ template.hasResourceProperties('Foo::Bar', {
 });
 ```
 
+You can also assert that the `Properties` section of all resources of type
+`Foo::Bar` contains the specified properties -
+
+```ts
+template.allResourcesProperties('Foo::Bar', {
+  Foo: 'Bar',
```

Comment:
`Foo: 'Bar'` is a property here, why not change the naming or the text preceding the example?